src/crawler/scraper.py

```diff
@@ -75,13 +77,19 @@ def __print_summary(self):
         table = tabulate(results_to_print, ['Score', 'Number Of Entries'])
         print(table)
 
+    def __sort_reviews(self):
+        if Reviews_order(self.reviews_order).name == 'MOST_RECENT':
+            return Sort.NEWEST      #get most recent reviews
+        elif Reviews_order(self.reviews_order).name == 'MOST_RELEVANT' :
+            return Sort.MOST_RELEVANT       #get most relevant reviews
```

Comment:
For clarity use Reviews_order(self.reviews_order) == Reviews_order.MOST_RECENT

Reply:
Have you verified that by changing the sort order we will get different results.

Reply:
> If no sort order is specified take Sort.MOST_RELEVANT as default value

If reviews_order has no value specified, then value will be assigned to "most relevant" . This is done in line 138 of main.py

Reply:
> Have you verified that by changing the sort order we will get different results.

Yes, I have verified and got different results for most recent and most relevant sort orders

Reply:
Looks good for me